src/reporters/CLIReporter.ts

```diff
@@ -30,6 +29,7 @@ export class CLIReporter implements IReporter {
         repoName = cwp.component.path;
       }
 
+      lines.push(bold(blue('----------------------------')));
```

Comment:
Is there a reason for moving this line?

Reply:
Yes, because the report is split by components and if the report has 1 component, this is a part of the top text "DX Scanner Result" and it's used as a separator between each separated component report.